### ISSUE TITLE
output makeflow filename duplicated?

### ISSUE THREAD

**Issue by nkern** [HERA-Team/hera_opm#4]
I'm running
```
build_makeflow_from_config.py -c abscal_pipe.cfg ./filename
```
and the output makeflow file has a filename
```
out.mf.abscal_pipe.cfg.mf
```
whereas the docstring says that by default it should be just `out.mf`. Is this extra `...abscal_pipe.cfg.mf` done on purpose? Or is this a bug?

**Comment by nkern** [comment]
yeah no problem! it seems like the easiest-to-understand convention would be
`build_makeflow_from_config.py -c config_file.cfg ./filename` yields
`config_file.mf`, but up to you!

**Comment by plaplant** [comment]
Sounds good, I'll add it!

**Comment by plaplant** [comment]
Fixed in fe873a6aeea249951eaaabfec2fe099a375c6795.
